File: Python/leetcode_1313.py
from typing import List


def decompress_rle_list(nums: List[int]) -> List[int]:
    result = []
    for i in range(0, len(nums) - 1, 2):
        result.extend([nums[i+1]] * nums[i])
    return result


print(decompress_rle_list([1, 1, 2, 3]))

File: JavaScript/leetcode_1313.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var decompressRLElist = function (nums) {
    let result = [];
    for (let index = 0; index < array.length - 1; index += 2) {
        result.push(...Array(nums[index]).fill(nums[i + 1]));
    }
    return result;
};
